DIFFERENCES.md

```diff
@@ -0,0 +1,82 @@
+This means that applications cannot depend on receiving a specific curve for any other value and may encounter compatibility issues should SunEC ever change its behavior or the application changes to a different JCE provider.
+ACCP removes this unspecified behavior by rejecting use of this method for all values not on the above list (for any ACCP version after 1.5.0) with an `InvalidParameterException`.
+
+For this reason, regardless of whether you use ACCP or not, we recommend that you only use the [KeyPairGenerator.initialize(AlgorithmParameterSpec params)](https://docs.oracle.com/javase/8/docs/api/java/security/KeyPairGenerator.html#initialize-java.security.spec.AlgorithmParameterSpec-) method with an [ECGenParameterSpec](https://docs.oracle.com/javase/8/docs/api/java/security/spec/ECGenParameterSpec.html) to generate EC keys.
+This construction is safe for all known JCE providers and is expected to remain safe even should providers change behavior in other ways.
+
+For more information, please see the [changelog](./CHANGELOG.md) notes for version 1.5.0.
+
+## Cipher.getOutputSize() for AES-GCM
+ACCP may overestimate the amount of space needed when encrypted with `AES/GCM/NoPadding`.
+While this is compliant with the JCE (which [permits overestimation](https://docs.oracle.com/javase/8/docs/api/javax/crypto/Cipher.html#getOutputSize-int-)) it has caused confusion for some developers.
+We are tracking this as [issue #135](https://github.com/corretto/amazon-corretto-crypto-provider/issues/135) and will improve this behavior.
+
+## SecureRandom is never deterministic
+Some implementation of `SecureRandom` (such as `SHA1PRNG`, provided by the default OpenJDK cryptographic providers) can operate deterministically if `SecureRandom.setSeed(byte[])` is called prior to any other methods.
+This behavior allows for insecure seeding and may result in lower security if the application requires the `SecureRandom` instance to provide secure entropy (such as for cryptographic use).
+The `SecureRandom` implementation provided by ACCP automatically seeds itself upon creation and cannot be used in a deterministic manner.
+This change is only relevant to systems which need deterministic behavior based on a seed such as some simulations.
+Systems which need deterministic behavior should not use an ACCP implementation of `SecureRandom` and should select an implementation/algorithm which specifically meets their needs.
+
+## SecureRandom uses thread local state internally
+To avoid the costs of both RNG initialization and thread contention, ACCP maintains a single internal instance of SecureRandom for each thread.
+Any time an instance of `SecureRandom` is used, ACCP will route the requests to the appropriate backing instance for the calling thread.
+As the output of calls to `SecureRandom` is computationally indistinguishable from actual random data, this implementation detail has no impact on callers (other than improving performance).
```

Comment:
```suggestion
Because the output of calls to `SecureRandom` is computationally indistinguishable from actual random data, this implementation detail has no impact on callers other than improving performance.
```